okhttp/src/main/java/com/squareup/okhttp/Cache.java

```diff
@@ -270,6 +270,19 @@ private void abortQuietly(DiskLruCache.Editor editor) {
   }
 
   /**
+   * Initialize the cache. This will include reading the journal files from
+   * the storage and building up the necessary in-memory cache information.
+   *
+   * The initialization time may vary depending on the journal file size and
+   * the current actual cache size. The application needs to be aware of calling
+   * this function during the initialization phase and preferrably in a background
+   * worker thread.
+   */
+  public void initialize() throws IOException {
+    cache.initialize();
```

Comment:
The method that you are calling here requires a lock on the `cache` object so this call would need surrounded with a `synchronized` block.